website/content/blog/2023-11-getting-started-as-a-tag-env-contributor.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting started as a TAG Environmental Sustainability Contributor
+description: Getting started as a TAG Environmental Sustainability Contributor
+slug: tbd
+weight: 1
+date:   2023-11-07 00:00:00 +0000
+author: Antonio Di Turi, Atreay Kukanur, Mfon Nta
+reviewers: Kristina Devochko, Leonard Pahlke, Marlow Weston
+---
+
+## Getting started as a TAG Environmental Sustainability Contributor
+
+<p class="mt-5 mb-5"><img src="/images/blogs/2023-11-getting-started-as-a-tag-environmental-sustainability-contributor/tag-env-logo.png" alt="TAG Environmental Sustainability Logo"></p>
+
+If you are reading this blog it might be that you are looking forward to starting contributing to sustainability in the cloud native world.  This document will help you, and you can start by becoming a member of the TAG or one of its associated working groups.
+
+Starting to contribute to open source and cloud native communities may often seem challenging and scary for beginners, but there are many ways you can embark on this journey. One way to join the community is to become a member of one of the Cloud Native Computing Foundation Technical Advisory Groups (CNCF TAG).
```

Comment:
```suggestion
Starting to contribute to open source and cloud native communities may often seem challenging and scary for beginners, but there are many ways you can embark on this journey. One way to engage with the community is to contribute to one of the Cloud Native Computing Foundation Technical Advisory Groups (CNCF TAG).
```

IMO if we say one option is, we should name more than just this one option :D 

1. join a TAG
2. contribute to a CNCF project (<-- this is the "normal" / "standard" way to contribute)

depending how you cut contribution you can also list things like organizing a cloud native themed meetup and all the other community focsued tasks but i would not mention this at this point.

Reply:
solved, check last commit